### PR TITLE
Feat: implement SpacesApi

### PR DESCRIPTION
**Pull request by smnandre**

### CODE REVIEW

File: tests/Unit/SpacesApiTest.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+/**
+ * This file is part of Storyblok-Api.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Storyblok\Api\Tests\Unit;
```
Comment (silasjoisten) [inline tests/Unit/SpacesApiTest.php]: I think we should put those tests into `SensioLabs\Storyblok\Api\Tests\Integration` namespace

File: README.md [hunk 2]
```diff
@@ -27,6 +27,31 @@ $client = new StoryblokClient(
...
+
+$client = new StoryblokClient(/* ... */);
+$spacesApi = new SpacesApi($client);
+
+$response = $spacesApi->me();
+
+// Example response
+$space = $response->getSpace();
+$space->getName(); // Example Space
+$space->getDomain(); // https:/example.com
```
Comment (OskarStark) [inline README.md]:
```suggestion
$space->getDomain(); // https://example.com
```

**Review by OskarStark — COMMENTED** [review]
Is there a need for us to retrieve the spaces info?

File: README.md (hunk 2 above)
Comment (silasjoisten) [reply README.md]: There is no getter anyway. docs are oudated

**Comment by silasjoisten** [comment]
> Is there a need for us to retrieve the spaces info?

yes there is to get the current cache version in order to the cds limitation and performance if we send a cache key in the url see: https://www.storyblok.com/docs/api/content-delivery/v2/getting-started/rate-limit

**Comment by silasjoisten** [comment]
@smnandre is not here today i am taking over.

**Pull request merged**